src/ModalCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import './App.css';
+
+export class ModalCard extends React.Component {
+    handleCloseModal = (event) => {
+        this.props.onCloseModal();
+
+        event.stopPropagation();
```

Comment:
any reason for this?

Reply:
Yes. If I do not stop the ascent, then extra handlers will work

Reply:
@DmitryApan did you test it?

Reply:
> @DmitryApan did you test it?

Yes